pkg/delete/persistence/delete_persistence.go

```diff
@@ -47,7 +47,8 @@ type DeleteEntry struct {
 	ConfigId string `yaml:"id,omitempty" json:"id,omitempty" mapstructure:"id" jsonschema:"description=The monaco ID of the config to be deleted - required for configs with generated IDs (e.g. Settings 2.0, Automations, Grail Buckets)."`
 	// ConfigName is the name of the config to be deleted - required for configs deleted by name (classic Config API types)
 	ConfigName string `yaml:"name,omitempty" json:"name,omitempty" mapstructure:"name" jsonschema:"description=The name of the config to be deleted - required for configs deleted by name (classic Config API types)."`
-
+	//OriginID is the dynatrace ID of the object
+	OriginID string `yaml:"objectId,omitempty" json:"objectId,omitempty" mapstructure:"objectId" jsonschema:"ID of the configuration in the Dynatrace. It can't combine with 'name'."`
```

Comment:
```suggestion
	OriginID string `yaml:"objectId,omitempty" json:"objectId,omitempty" mapstructure:"objectId" jsonschema:"ID of the configuration in the Dynatrace. It can't be combined with 'name'."`
```